dist/js/foundation.d.ts

```diff
@@ -73,6 +67,10 @@ declare module FoundationSites {
     destroy(): void;
   }
 
+  interface AccordionDefaults extends IAccordionOptions{
```

Comment:
Empty interface? So why not reference IAccordionOptions directly. From my look on the JS I'm also not sure if Defaults and options are the same. From what I see they are not.